test/benchmark/__init__.py

```diff
@@ -1,127 +1,7 @@
-            return t.dif
-
-        def main(self):
-            for game in sorted(AutoWorld.AutoWorldRegister.world_types):
-                summary_data: typing.Dict[str, collections.Counter[str]] = {
-                    "empty_state": collections.Counter(),
-                    "all_state": collections.Counter(),
-                }
-                try:
-                    multiworld = MultiWorld(1)
-                    multiworld.game[1] = game
-                    multiworld.player_name = {1: "Tester"}
-                    multiworld.set_seed(0)
-                    multiworld.state = CollectionState(multiworld)
-                    args = argparse.Namespace()
-                    for name, option in AutoWorld.AutoWorldRegister.world_types[game].options_dataclass.type_hints.items():
-                        setattr(args, name, {
-                            1: option.from_any(getattr(option, "default"))
-                        })
-                    multiworld.set_options(args)
-
-                    gc.collect()
-                    for step in self.gen_steps:
-                        with TimeIt(f"{game} step {step}", logger):
-                            call_all(multiworld, step)
-                            gc.collect()
-
-                    locations = sorted(multiworld.get_unfilled_locations())
-                    if not locations:
-                        continue
-
-                    all_state = multiworld.get_all_state(False)
-                    for location in locations:
-                        time_taken = self.location_test(location, multiworld.state, "empty_state")
-                        summary_data["empty_state"][location.name] = time_taken
-
-                        time_taken = self.location_test(location, all_state, "all_state")
-                        summary_data["all_state"][location.name] = time_taken
-
-                    total_empty_state = sum(summary_data["empty_state"].values())
-                    total_all_state = sum(summary_data["all_state"].values())
-
-                    logger.info(f"{game} took {total_empty_state/len(locations):.4f} "
-                                f"seconds per location in empty_state and {total_all_state/len(locations):.4f} "
-                                f"in all_state. (all times summed for {self.rule_iterations} runs.)")
-                    logger.info(f"Top times in empty_state:\n"
-                                f"{self.format_times_from_counter(summary_data['empty_state'])}")
-                    logger.info(f"Top times in all_state:\n"
-                                f"{self.format_times_from_counter(summary_data['all_state'])}")
-
-                except Exception as e:
-                    logger.exception(e)
-
-    runner = BenchmarkRunner()
-    runner.main()
+    import path_change
+    path_change.change_home()
+    import load_worlds
+    load_worlds.run_load_worlds_benchmark()
+    import locations
+    locations.run_locations_benchmark()
```

Comment:
besides the missing newline at end of file, i think to avoid linter warnings, this should be changed to

```python
def run():
    import ...
    ...

run()
```

or maybe even splitting into init() and run() where run only happens for `__name__ == "__main__"` (also see next comment)